### DISCUSSION TITLE
[Support] 增加 casbin-mikro-adapter 实现

### DISCUSSION THREAD

**Discussion by hooliy** [midwayjs/midway#3721] [Q&A]
### Describe the problem（描述问题）

方便实现一个 mikro-orm 的 casbin adapter吗？ typeorm官方的bug太多了，解决很不及时。建议实现 casbin-mikro-adapter

### Midway Versions（Midway 版本）

latest

**Comment by czy88840616** [comment]
我觉得最好的实现。。其实是 redis。。

**Reply by hooliy** [reply]
刚接触 casbin ，业务系统的用户、角色、权限多表关系是不是与 policy 没有关系？policy不需要持久化到数据库中吗？

**Reply by hooliy** [reply]
是不是可以理解为：业务系统中的用户权限发生变化 ，重新生成 policy 更新到redis中就可以了？

**Reply by czy88840616** [reply]
是这样的

**Reply by hooliy** [reply]
好的，谢谢！
